bundles/org.openhab.binding.netatmo/README.md

```diff
@@ -581,111 +483,69 @@ All these channels except welcomePersonAtHome are read only.
 // Bridge configuration:
 Bridge netatmo:netatmoapi:home "Netatmo API" [ clientId="*********", clientSecret="**********", username = "mail@example.com", password = "******", readStation=true, readThermostat=false] {
     // Thing configuration:
-    Thing NAMain inside "Netatmo Inside"       [ id="aa:aa:aa:aa:aa:aa" ]
-    Thing NAModule1 outside "Netatmo Outside"  [ id="bb:bb:bb:bb:bb:bb", parentId="aa:aa:aa:aa:aa:aa" ]
-    Thing NAModule3 rain "Netatmo Rain"        [ id="cc:cc:cc:cc:cc:cc", parentId="aa:aa:aa:aa:aa:aa" ]
+    Bridge netatmo:NAMain:inside "Netatmo Inside"       [ id="aa:aa:aa:aa:aa:aa" ] {
+        NAModule1 outside "Netatmo Outside"  [ id="bb:bb:bb:bb:bb:bb" ]
+        NAModule3 rain "Netatmo Rain"        [ id="cc:cc:cc:cc:cc:cc" ]
+    }
 }
```

Comment:
Suggest to also add example with Channels.

Reply:
Done.

Reply:
@clinique - would it be possible to add an example with those dynamically added channels, e.g. measurement channels - from files? I still didn't figure it out. Or maybe just quick example for me here. As soon as I'm able to configure this I can back to full "production" testing.